### PR TITLE
Add car crash scenario.

### PR DESCRIPTION
**Pull request by Extension-Driver**
#### Summary
Content "Add the car crash scenario"

**MINIMUM VIABLE PRODUCT**
- [x] Add basic scenario definition
- [x] Check player spawns with wounds.
- [x] Find out where the roads are defined in the game files & add them.

#### Purpose of change

Revives #47362.

We have a helicopter crash scenario already; that is limited to military characters. This allows for every character to start in a wreck if the player wishes them to. I don;t think a scenario like this has been done before.

Besides, the idea of escaping the apocalypse by car and failing is a common trope in apocalypse films.

#### Describe the solution

Add a car crash scenario to the game. 

#### Describe alternatives iyou've considered
None!

#### Testing
Load in game, check if it works.

#### Additional context
If I could get this to spawn zombies and other random items around the crash site it would be great.

### CODE REVIEW

File: data/json/scenarios.json [hunk 1]
```diff
@@ -574,5 +574,16 @@
     "allowed_locs": [ "sloc_private_resort" ],
     "professions": [ "frat", "gangster", "lost_sub", "lawyer", "politician", "dancer" ],
     "flags": [ "CHALLENGE", "LONE_START" ]
+  },
+  {
+    "type": "scenario",
+    "id": "car_crash",
+    "name": "Car Crash",
+    "points": -2,
+    "description": "You decided the open road would be a better prospect than the zombie-filled cities.  While en route to your destination, you were involved in a wreck. Injured and alone, death seems inevitable.",
```
Comment (Night-Pryanik) [inline data/json/scenarios.json:583]:
```suggestion
    "description": "You decided the open road would be a better prospect than the zombie-filled cities.  While en route to your destination, you were involved in a wreck.  Injured and alone, death seems inevitable.",
```

**Comment by actual-nh** [comment]
Should this be flagged as "CHALLENGE"?

**Comment by Extension-Driver** [comment]
What would that do? AFAICT that doesn't do anything...

**Comment by Night-Pryanik** [comment]
> What would that do? AFAICT that doesn't do anything...

This flag will prevent this scenario from being selected when player chooses random game modes.

**Comment by Extension-Driver** [comment]
Yeah, that flag's set. It's all done now.

**Pull request merged**